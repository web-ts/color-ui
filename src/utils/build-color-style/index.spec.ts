import { describe, expect, it } from "vitest";

import buildColorStyle from "./";

describe("buildColorStyle", () => {
  it("should return the correct style", () => {
    expect(buildColorStyle("red", "tertiary")).toMatchSnapshot();
  });
});
