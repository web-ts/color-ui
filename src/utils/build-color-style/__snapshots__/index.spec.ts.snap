// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`buildColorStyle > should return the correct style 1`] = `"--red-base: var(--color-tertiary-base); --red-base-text: var(--color-tertiary-base-text); --red-light: var(--color-tertiary-light); --red-light-text: var(--color-tertiary-light-text); --red-lighter: var(--color-tertiary-lighter); --red-lighter-text: var(--color-tertiary-lighter-text); --red-lightest: var(--color-tertiary-lightest); --red-lightest-text: var(--color-tertiary-lightest-text); --red-dark: var(--color-tertiary-dark); --red-dark-text: var(--color-tertiary-dark-text); --red-darker: var(--color-tertiary-darker); --red-darker-text: var(--color-tertiary-darker-text); --red-darkest: var(--color-tertiary-darkest); --red-darkest-text: var(--color-tertiary-darkest-text)"`;
